platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/network/OutboundConnectionCreatorTest.java

```diff
@@ -99,16 +99,17 @@ void createConnectionTest() throws IOException, ConstructableRegistryException {
         final SocketFactory socketFactory = mock(SocketFactory.class);
         doAnswer(i -> socket).when(socketFactory).createClientSocket(any(), anyInt());
 
-        final SocketConfig socketConfig = getSocketConfig();
-
+        final SocketConfig socketConfig = getConfig().getConfigData(SocketConfig.class);
+        final Configuration configuration = new TestConfigBuilder().getOrCreateConfig();
```

Comment:
there exists in this file the `getConfig` method. Any reason not to use it here?

Reply:
Added, thanks!